src/Symfony/Bridge/Monolog/Tests/LoggerTest.php

```diff
@@ -32,7 +32,7 @@ public function testGetLogsWithDebugHandler()
 
     public function testGetLogsWithoutDebugHandler()
     {
-        $handler = $this->getMock('Symfony\Component\HttpKernel\Log\LoggerInterface');
+        $handler = $this->getMock('Psr\Log\LoggerInterface');
```

Comment:
this makes no sense. (The old test does not either btw). This should be a Monolog handler mock